### PR TITLE
feat: notebook function reference

### PR DESCRIPTION
**Pull request by drdelambre**
Peek 2021-01-22 15-10

This migrates the function reference functionality from data explorer to notebooks. not a perfect 1:1 from a product perspective, but close enough to serve as a stand in to scratch the itch while we have talks about better ways to do it

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 57 | :red_circle:
accessibility | 65 | :yellow_circle:
best-practices | 87 | :green_circle:
seo | 80 | :yellow_circle:
pwa | null | :question:

File: src/flows/pipes/RawFluxEditor/view.tsx [hunk 1]
```diff
@@ -44,27 +52,97 @@ const Query: FC<PipeProp> = ({Context}) => {
...
+    />
+  )
+
+  return (
+    <Context controls={controls}>
+      <Suspense
+        fallback={
+          <SpinnerContainer
+            loading={RemoteDataState.Loading}
+            spinnerComponent={<TechnoSpinner />}
```
Comment (genehynson) [inline src/flows/pipes/RawFluxEditor/view.tsx:132]: Potentially replace with `<PageSpinner/>`?

File: src/flows/pipes/RawFluxEditor/view.tsx [hunk 2]
```diff
@@ -44,27 +52,97 @@ const Query: FC<PipeProp> = ({Context}) => {
...
-        </Suspense>
-      </Context>
-    ),
-    [query.text, updateText]
+        }
+      >
+        <FluxMonacoEditor
+          script={query.text}
+          onChangeScript={updateText}
+          onSubmitScript={() => {}}
```
Comment (genehynson) [inline src/flows/pipes/RawFluxEditor/view.tsx:139]: why empty function for onSubmitScript?

**Review by genehynson — APPROVED** [review]
lgtm 🎉  Left a couple nit questions

File: src/flows/pipes/RawFluxEditor/view.tsx (hunk 1 above)
Comment (drdelambre) [reply src/flows/pipes/RawFluxEditor/view.tsx:132]: i'm confused. DMs INCOMING

File: src/flows/pipes/RawFluxEditor/view.tsx (hunk 2 above)
Comment (drdelambre) [reply src/flows/pipes/RawFluxEditor/view.tsx:139]: no idea! should totally be an optional parameter

File: src/flows/pipes/RawFluxEditor/view.tsx (hunk 1 above)
Comment (genehynson) [reply src/flows/pipes/RawFluxEditor/view.tsx:132]: to clarify for future readers: 
```
<SpinnerContainer
loading={RemoteDataState.Loading}
spinnerComponent={<TechnoSpinner />}
/>
 ```
===
```
<PageSpinner/>
```

Comment (drdelambre) [reply src/flows/pipes/RawFluxEditor/view.tsx:132]: captured the work in #555 cause it seems like we'd want to normalize this across the app

**Pull request merged**